scripts/pylib/twister/twisterlib.py

```diff
@@ -3766,7 +3775,7 @@ def xunit_report(self, filename, platform=None, full_report=False, append=False,
     def csv_report(self, filename):
         with open(filename, "wt") as csvfile:
             fieldnames = ["test", "arch", "platform", "status",
-                          "extra_args", "handler", "handler_time", "ram_size",
+                          "extra_args", "extra_bin_args", "handler", "handler_time", "ram_size",
```

Comment:
this is specific to one class of handlers, what happens when the next person comes and wants to add arguments to another type of handler? IMO this should be made generic to all handlers.

Reply:
Hmm, ok, yeah I guess you are right. In what way would you prefer me to change this? Is it just mainly the naming and making sure that it's set for all types of handlers? But then not using it in the other handlers, except in the case of `BinaryHandler`?

Also, I guess I should check for native type in the `BinaryHandler` also when applying the args as I do? Or should we have some other syntax in the yaml for the arguments to indicate that "in this testcase, these arguments are meant for notive_posix"?

Reply:
> Hmm, ok, yeah I guess you are right. In what way would you prefer me to change this? Is it just mainly the naming and making sure that it's set for all types of handlers? But then not using it in the other handlers, except in the case of `BinaryHandler`?

naming, but then also make it apply to more than just the binary handler. I am fine if this initially takes effect on native_posix only with proper documentation and we can add the others later.

Reply:
OK, I will try to do a fix for this, thanks for the input.